.github/workflows/docker-ghcr.yml

```
@@ -0,0 +1,25 @@
+name: Publish a Docker image to ghcr.io
+on:
+  pull_request:
+    branches:
+      - main
```

I will change the trigger if I can confirm that this action works as expected.

```suggestion
  push:
    tags:
      - v*.*.*
```

---

This action successfully published a Docker image.

* https://github.com/optuna/optuna-dashboard/pkgs/container/optuna-dashboard
* https://github.com/optuna/optuna-dashboard/actions/runs/1631332945